src/actions/GetText.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  Action,
+  ActionPreferences,
+  ActionReturnValue,
+  Graph,
+  TypedSelection,
+} from "../Types";
+import displayPendingEditDecorations from "../editDisplayUtils";
+
+export default class GetText implements Action {
```

Comment:
This one will be great to have, eg for "phones"

Reply:
Yes phones and formatters. I find myself missing both.